docs/website/docs/plus/getting-started/installation.md

```diff
@@ -51,6 +54,7 @@ On macOS, you can use [Homebrew](https://brew.sh) to install Python 3.10.
 ```sh
 brew update
 brew install python@3.10
+python3.10 -m ensurepip --upgrade
```

Comment:
usually python should be available as the `python` command and you can use `pyenv` to manage multiple python versions and which one is active. Or is there a specific reason you add the python command with the version here?

Reply:
@sh-rp the instructions came with a specific version range  (Python 3.9-3.13) so I just made sure all commands used the newly installed python rather than a potentially lower version